src/repos/PublicationRepo.ts

```diff
@@ -3,6 +3,10 @@ import { Publication, PublicationModel } from '../entities/Publication';
 
 import publicationsJSON from '../../publications.json';
 
+function getImageURL(slug: string, type: string): string {
+  return `${process.env.IMAGE_PATH}/${slug}/${type}.png`;
+}
+
```

Comment:
can we add to the documentation in either `assets/volume` or in this repo that all images must be of type `.png`? I'm sure whoever is going to add images in the future will likely assume this much by looking at existing content, but we should still get in the happen of connecting niche business logic like this with some form of documentation for onboarding new backend devs once you hand this over to someone else

Reply:
also, more Volume specific but adding documentation on the type of image, 'profile' or 'background', we may add more assets for publications in the future depending on how UI changes?